docs/devel/coding-conventions.md

```diff
@@ -30,12 +30,38 @@ Documentation for other releases can be found at
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-Coding style advice for contributors
+Code conventions
   - Bash
     - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
   - Go
     - https://github.com/golang/go/wiki/CodeReviewComments
-    - https://gist.github.com/lavalamp/4bd23295a9f32706a48f
+    - Command-line flags should use dashes, not underscores
+    - API conventions
+      - [API changes](api_changes.md)
+      - [API conventions](api-conventions.md)
+    - [Kubectl conventions](kubectl-conventions.md)
```

Comment:
It seems like all of the above could be some form of static analysis tools prior to PR.